### DISCUSSION TITLE
How do I know if a project is paused from a API?

### DISCUSSION THREAD

**Discussion by ruggi99** [supabase/supabase#11431] [Feature Requests]
### Discussed in https://github.com/supabase/supabase/discussions/11267

<div type='discussions-op-text'>

<sup>Originally posted by **ruggi99** December 26, 2022</sup>
Hi all,
haven't found an answer elsewhere.

Is there something in the API that tells me if a project is paused due to inactivity?
My app seems buggy when a project is paused because it waits indefinetely for a response (via websocket) from Supabase, which doesn't come.
If there's nothing I will change this into a feature request.

Thanks</div>

**Comment by saltcod** [comment]
Hi @ruggi99 — I think you're right. I flagged this with the CLI team internally & I'm going to turn it into a feature request now. Thank you!
